prisma/schema.sql

```diff
@@ -0,0 +1,158 @@
+CREATE TYPE provinces AS ENUM('NL', 'PE', 'NS', 'NB', 'QC', 'ON', 'MB', 'SK', 'AB', 'BC', 'YT', 'NT', 'NU');
+CREATE TYPE weekdays AS ENUM ('MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN');
+
+-- npx prisma introspect && npx prisma generate
+-- Create users table
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255),
+  email VARCHAR(255) UNIQUE,
+  email_verified TIMESTAMPTZ,
+  image TEXT,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create email verification requests table
+CREATE TABLE verification_requests (
+  id SERIAL PRIMARY KEY NOT NULL,
+  identifier VARCHAR(255) NOT NULL,
+  token VARCHAR(255) NOT NULL UNIQUE,
+  expires TIMESTAMPTZ NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE program (
```

Comment:
For documentation standard, please add comments on top. `-- Create program table`

Reply:
Same for other tables too!

Reply:
should we add a `created_at` and `updated_at` field to this table as well as all other table for tracking purposes?

Reply:
Yea we should. I thought Prisma would auto generate those ones actually but perhaps I remembered incorrectly